src/axom/quest/CMakeLists.txt

```diff
@@ -83,19 +80,25 @@ set( quest_depends_on
      fmt
      )
 
-blt_list_append(TO quest_depends_on IF SPARSEHASH_FOUND ELEMENTS sparsehash)
-blt_list_append(TO quest_depends_on IF ENABLE_OPENMP ELEMENTS openmp)
 blt_list_append(TO quest_depends_on IF ENABLE_CUDA ELEMENTS cuda)
+blt_list_append(TO quest_depends_on IF ENABLE_OPENMP ELEMENTS openmp)
+blt_list_append(TO quest_depends_on IF SPARSEHASH_FOUND ELEMENTS sparsehash)
+     
+if(C2C_FOUND)
+     list(APPEND quest_headers readers/C2CReader.hpp)
+     list(APPEND quest_sources readers/C2CReader.cpp)
+     list(APPEND quest_depends_on c2c)
+endif()
 
 if (ENABLE_MPI)
-    list(APPEND quest_headers stl/PSTLReader.hpp)
-    list(APPEND quest_sources stl/PSTLReader.cpp)
-
-    blt_list_append(TO quest_depends_on
-                    IF AXOM_ENABLE_LUMBERJACK
-                    ELEMENTS lumberjack )
+    list(APPEND quest_headers readers/PSTLReader.hpp)
+    blt_list_append(TO quest_headers IF C2C_FOUND ELEMENTS readers/PC2CReader.hpp)
 
+    list(APPEND quest_sources readers/PSTLReader.cpp)
+    blt_list_append(TO quest_sources IF C2C_FOUND ELEMENTS readers/PC2CReader.cpp)
+    
     list(APPEND quest_depends_on mpi)
+    blt_list_append(TO quest_depends_on IF AXOM_ENABLE_LUMBERJACK ELEMENTS lumberjack )
```

Comment:
This doesn't technically need to be here since it will be inherited by SLIC. (minus something I cant think of)

Reply:
Thanks -- removed.